src/rascal/utils.hh

```diff
@@ -38,6 +38,63 @@
 namespace rascal {
   namespace internal {
 
+    namespace details {
+      /**
+       * Recursivelly go through a list of Orders and check if at least one
+       * matches TargetOrder at compile time.
+       */
+      template <size_t TargetOrder, size_t Order, size_t... Orders>
+      struct IsOrderAvailableHelper {
+        static constexpr bool is_order_available = (TargetOrder == Order);
+
+        template <bool T = is_order_available, std::enable_if_t<T, int> = 0>
```

Comment:
same with `bool T`

Reply:
And why is this constexpr using snake_case, while the other one is using CamelCase? I don't care which one is used, but there should only be one.

Reply:
rename a few static constexpr CamelCase to snake_case